kt/godot-library/src/main/kotlin/godot/gen/godot/AnimationNodeOneShot.kt

```diff
@@ -22,102 +22,57 @@ import kotlin.Long
 import kotlin.Suppress
 
 /**
- * Plays an animation once in an [godot.AnimationNodeBlendTree].
- *
- * Tutorials:
- * [https://godotengine.org/asset-library/asset/678](https://godotengine.org/asset-library/asset/678)
- *
- * A resource to add to an [godot.AnimationNodeBlendTree]. This animation node will execute a sub-animation and return once it finishes. Blend times for fading in and out can be customized, as well as filters.
- *
- * After setting the request and changing the animation playback, the one-shot node automatically clears the request on the next process frame by setting its `request` value to [ONE_SHOT_REQUEST_NONE].
- *
- * [codeblocks]
- *
- * [gdscript]
- *
+ * A resource to add to an [AnimationNodeBlendTree]. This animation node will execute a
+ * sub-animation and return once it finishes. Blend times for fading in and out can be customized, as
+ * well as filters.
+ * After setting the request and changing the animation playback, the one-shot node automatically
+ * clears the request on the next process frame by setting its `request` value to [ONESHOTREQUESTNONE].
```

Comment:
`ONESHOTREQUESTNONE` something is off here. this should be `ONE_SHOT_REQUEST_NONE`